GNUmakefile

```diff
@@ -1,5 +1,6 @@
 VERSION=v1.1.5
 
+SWEEP?=us-east-1
```

Comment:
`us-east-1`?

was this stolen from an AWS provider or something? :)

Reply:
Yea just a dumby value to enable sweeping. They had no regionless implementation of the sweepers so I just hardcode one and ignore it in the sweep functions.